numalogic/tools/callbacks.py

```diff
@@ -43,4 +43,9 @@ def on_train_epoch_end(self, trainer: pl.Trainer, pl_module: pl.LightningModule)
         metrics = self.get_metrics(trainer, pl_module)
         curr_epoch = trainer.current_epoch
         if curr_epoch % self._log_freq == 0:
-            _LOGGER.info("epoch %s, loss: %s", curr_epoch, metrics["loss"])
+            _LOGGER.info("epoch=%s, training_loss=%s", curr_epoch, metrics["loss"])
```

Comment:
Do we need to log additional context, such as which model is this loss info about?

Reply:
I would argue not here, because `enable_mode_summary` flag prints out the model architecture anyway. This is strictly for epoch logs.